src/test/java/com/github/mauricioaniche/ck/realworld/RealWorldClassesTest.java

```diff
@@ -232,4 +233,15 @@ void ambry_ByteBufferAsyncWritableChannel() {
 		Assertions.assertNotNull(class1.getMethod("resolveChunk/1[java.lang.Exception]"));
 	}
 
+	@Test
```

Comment:
Would you mind just adding a comment here, explaining that this test was related to issue #65?

Reply:
Ok, I will do that

Reply:
Done